src/Pages/MainPage/Containers/MainContainer.tsx

```diff
@@ -52,18 +54,48 @@ const MainContainer: React.FC = () => {
     setEditingTaskId(null);
   };
 
-  const sortedTasks = tasks.slice().sort((a, b) => {
-    if (a.completed && !b.completed) return 1;
-    if (!a.completed && b.completed) return -1;
-    return 0;
-  });
+  const applyFilter = (tasks: Task[], filter: string): Task[] => {
```

Comment:
Should also add cases where users can view only the high or medium or low priority tasks.

Reply:
Pushed a new commit